terraso_backend/apps/project_management/models/sites.py

```diff
@@ -38,6 +39,9 @@ class Meta(BaseModel.Meta):
     name = models.CharField(max_length=200)
     latitude = models.FloatField()
     longitude = models.FloatField()
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
```

Comment:
question: i thought django generate these fields automatically, is that not accurate?

Reply:
I thought so to, but this is actually done by the parent class we are extending. Removed the extra fields.